lms/djangoapps/student_account/views.py

```diff
@@ -421,6 +425,13 @@ def account_settings_context(request):
     user = request.user
 
     year_of_birth_options = [(unicode(year), unicode(year)) for year in UserProfile.VALID_YEARS]
+    try:
+        user_orders = get_user_orders(user)
+    except Exception as err:  # pylint: disable=broad-except
+        log.exception('Error fetching order history from otto: %s', err.message)
```

Comment:
@ahsan-ul-haq if you're using `log.exception`, you don't need to include the error message here; it will be included in the stacktrace. I would just do:
```py
try:
    user_orders = get_user_orders(user)
except:  # pylint: disable=bare-except
    log.exception('Error fetching order history from Otto.')
```